.github/buildomat/build-and-test.sh

```diff
@@ -71,6 +71,11 @@ ptime -m timeout 1h cargo test --doc --locked --verbose --no-fail-fast
 # to check is to try to remove it with `rmdir`.
 #
 unset TMPDIR
+
+# We expect the seed CRDB to be placed here, so we explicitly remove it. There
+# is no "final test", so cargo test wouldn't really know when to remove it.
+rm -rf "$TEST_TMPDIR/crdb-base"
```

Comment:
This data used to be placed in:

> OUT_DIR — the folder in which all output and intermediate artifacts should be placed. This folder is inside the build directory for the package being built, and it is unique for the package in question.

(Which is an environment variable cargo sets for build scripts)

Well, this isn't run by a build script any longer, so now it's going in `$TEST_TMPDIR`, which is why I'm removing it explicitly now.